test/selenium/signInPage.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var SignedInPageView = require('./lib/view/signed_in/view.js');
+var signedInPageView;
+
+function SignInPage(driver) {
+    this.driver = driver;
+    signedInPageView = new SignedInPageView(this.driver);
+}
+
+SignInPage.prototype = {
+    openPage : function() {
```

Comment:
Nit: There's no real action done by the user, so `open` is a bit misleading. What do you think of `getSignedInPage` ?

*No need to make the following change now:* On another note, I think this class has to be in charge of initializing the right view. For example:
``` js
landOnSignUp: function() {
  // clear cookies and return the SignUpView
},

landOnSignedIn: function() {
  // POST a request to get a token and return the SignInView
}

// And so on
```
What do you think?